src/IRCSock.cpp

```diff
@@ -970,9 +970,31 @@ bool CIRCSock::OnPrivMsg(CNick& Nick, CString& sMessage) {
 	return false;
 }
 
+// #124: OnChanMsg(): nick doesn't have perms
+static void FixupChanNick(CNick& Nick, CChan* pChan) {
```

Comment:
needs write access to the chan nick (to sync ident & host), but const overload of FindNick() would return const CNick*